### PR TITLE
fix(cmd): Split old `add` command into `new` and `add`.

### PR DESCRIPTION
**Pull request by dustmop**

### CODE REVIEW

**Comment by b5** [comment]
any idea y tests r fail? I'll have a look at code now.

File: cmd/add.go [hunk 1]
```diff
@@ -1,82 +1,40 @@
...
-
-Once you’ve added data, you can use the export command to pull the data out of 
-qri, change the data outside of qri, and use the save command to record those 
-changes to qri.`,
-		Example: `  add a new dataset named annual_pop:
-  $ qri add --body data.csv me/annual_pop
-
-create a dataset with a dataset data file:
-  $ qri add --file dataset.yaml --body comics.csv me/comic_characters`,
+Add retrieves a dataset owned by another peer and adds it to your repo.`,
```
Comment (b5) [inline cmd/add.go]: muuuch cleaner.

File: cmd/new.go [hunk 2]
```diff
@@ -0,0 +1,200 @@
...
+
+
+const providingSecretWarningMessage = `
+Warning: You are providing secrets to a dataset transformation.
+Never provide secrets to a transformation you do not trust.
+continue?`
+
+
+// NewNewCommand creates a new command
+func NewNewCommand(f Factory, ioStreams IOStreams) *cobra.Command {
```
Comment (b5) [inline cmd/new.go]: lol `NewNewCommand`

**Comment by dustmop** [comment]
> any idea y tests r fail? I'll have a look at code now.

Ack, something I need to dig into. We're using cobra.Command.ExecuteC, which I believes uses os.Exit(1) when it fails, meaning there's no error message reporting at all. I'll try and replace that before going forward with this.

**Comment by b5** [comment]
That would be delightful. I've come across similar issues that came when we switched away from `panic` to `os.Exit`. You're much better at determining what should happen in this context, so if you need to change either the behavior of qri (bring back panic), _or_ the tests, feel free. I'd like to do whichever feels more "Unix idiomatic"

**Pull request merged**